R/paste.R

```diff
@@ -0,0 +1,24 @@
+#' "Paste" together factors.
+#'
+#' Order is cartesian product of all levels.
+#'
+#' This is sometimes useful when plotting a factor
```

Comment:
The third sentence goes into "Details", is this intended? How is your implementation different from base::interaction()?